config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root "articles#index"
-  resources :articles
+  resources :articles do
+    resources :comments, only: [:create, :index]
```

Comment:
The comments controller doesn't have an index action, so I would either remove this index route or add a corresponding action and view.